### ISSUE TITLE
Help with commands

### ISSUE THREAD

**Issue by jeffpaterson** [luisiam/homebridge-liftmaster2#8]
"Open the garage door", "Open the garage", and "Open the garage door opener", all work well.

However, Siri will intercept anything short of "Close the garage door opener", and show me nearby bars with "garage" in their name. "Close the garage door opener" doesn't sound very natural.

Any suggestions? Also, is there an overview of available commands?

**Comment by luisiam** [comment]
I never use Siri to open or close the garage door. How about setting a scene?

**Comment by tekuonline** [comment]
@jeffpaterson Here is the list of siri commands [https://hey-siri.io](url).
In your case, creating a scene might help "Close the garage door opener", as @luisiam suggested.

**Comment by tommyd75** [comment]
Can you go into the device in the home app and see if it's named "garage door" or "garage door opener" 
I'm using a different plugin with my garage door and I changed my name to be "garage door" and my open and close work with just  Open the garage door or close the garage door.

**Comment by dmzimmerman** [comment]
I named my two openers "small garage door" and "big garage door" and Siri works fine with both of them as far as I've been able to tell.

**Comment by michaelwhill** [comment]
trying to switch to liftmaster2 from liftmaster.  I have two garage door openers and in liftmaster i created an accessory that included the login and deviceID that corresponded to the garage door identifier.  It sounds like dmzimmerman found a way to utilize two doors but I am unsure if we just add the property deviceID to the platform entry or create accessories similar to liftmaster(1).

**Comment by luisiam** [comment]
This plugin should be able to detect and add all your garage door in your account automatically. You do not need to specify any device ID.

**Comment by dmzimmerman** [comment]
Indeed, that's exactly what it did for me when I migrated from liftmaster to this plugin. I then renamed them in the Home app, to allow Siri to control them with reasonable names.

**Comment by iRayanKhan** [comment]
Just say "Hey Siri, Close the (Double) Garage

**Comment by iRayanKhan** [comment]
This issue seems to be stale and I may have necro replied. @luisiam Close this?
